app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/download_lloyd_george_workflow.cy.js

```diff
@@ -43,27 +84,116 @@ describe('GP Workflow: View Lloyd George record', () => {
                     body: baseUrl + '/browserconfig.xml', // uses public served file in place of a ZIP file
                 }).as('documentManifest');
 
+                cy.intercept('GET', '/SearchDocumentReferences*', {
+                    statusCode: 200,
+                    body: [
+                        {
+                            fileName: '1of2_testy_test.pdf',
+                            created: '2024-05-07T14:52:00.827602Z',
+                            virusScannerResult: 'Clean',
+                            id: 'test-id',
+                        },
+                        {
+                            fileName: '2of2_testy_test.pdf',
+                            created: '2024-05-07T14:52:00.827602Z',
+                            virusScannerResult: 'Clean',
+                            id: 'test-id-2',
+                        },
+                        {
+                            fileName: '3of2_testy_test.pdf',
```

Comment:
I know it does not effect the test but to reduce confusion down the line, can we change this to the correct format 1of3 ect..